code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -237,14 +240,18 @@
 		. = TRUE //no afterattack
 		if(machine_stat & (NOPOWER|BROKEN))
 			return
+		var/obj/item/reagent_containers/beaker_out
 		if(beaker)
-			to_chat(user, "<span class='warning'>A container is already loaded into [src]!</span>")
-			return
+			beaker_out = eject_beaker() //now with 100% more swapping
 		if(!user.transferItemToLoc(I, src))
 			return
-
+		if(beaker_out)
+			if(user && Adjacent(user) && user.can_hold_items())
+				user.put_in_hands(beaker_out)
```

Comment:
isn't this already handled by the try_put_in_hands in eject_beaker?

Reply:
So. Its very likely that I am stupid, but when I tried switching it out for just eject_beaker().
I found myself running into issues where it started to drop stuff at my feet instead.
Even if it should have been able to go into my hand.

Reply:
look at 246 which occurs after eject_beaker

Reply:
You're not stupid, don't walk into that sort of thing.

It's most likely because it's trying to transfer the beaker while the one in your hands is still well, in your hands. The order ought to be

Remove beaker from machine, store in temporary var
Put beaker in your hands into the machine
Put beaker in temporary var into your hands

Not sure if there's a clean way to do that though, outside of replacing this check with another try_put_in_hand(), which is a temporary measure